src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/IDebugProfileLaunchTargetsProvider3.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Debug
+{
+    /// <summary>
+    /// Optional interface that is used to tell whether a project should appear in
+    /// the Startup drop down list.
+    /// </summary>
+    internal interface IDebugProfileLaunchTargetsProvider3
```

Comment:
This should probably be public to match the '1' and '2' versions of the interface and allow extenders to implement it.

Reply:
done